x-pack/docs/en/rest-api/security/grant-api-keys.asciidoc

```diff
@@ -0,0 +1,133 @@
+[role="xpack"]
+[[security-api-grant-api-key]]
+=== Grant API key API
+++++
+<titleabbrev>Grant API keys</titleabbrev>
+++++
+
+Creates an API key on behalf of another user for access without requiring basic
+authentication.
```

Comment:
I think this description could be misinterpreted. It's based on the description on `Create API key` but the addtion of `on behalf of another user` means it might be parsed in 2 ways:

- On behalf of another user, Create an API key that allows access without requiring basic authentication _(the correct understanding)_
- Create an API key for access on behalf of another user, without requiring the user's authentication _(incorrect)_.

I can't find a substantially better description without dropping some part. I wonder whether we can avoid describing "API keys" here, and  and just say:

```suggestion
Creates an API key on behalf of another user.
```

Reply:
Thanks, I've made that change in the JSON file too